embedded/src/main/java/ui/controller/Controller.java

```diff
@@ -0,0 +1,153 @@
+package ui.controller;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Date;
+import java.util.Map;
+import java.util.concurrent.*;
+
+public class Controller {
+
+    private static final ScheduledExecutorService EXECUTOR_SERVICE = Executors.newSingleThreadScheduledExecutor();
+
+    private static final String WASHING_STATION_ID = "8";
```

Comment:
Shouldn't we pass this as an argument when starting up the embedded app?

Reply:
will add at the end